README.md

```diff
@@ -1 +1,14 @@
-# beyond08-group3
\ No newline at end of file
+					buyTogether‏
+
+
+
+	Idea 
+the idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.
+
+
+	Goals 
+the goals of the project are:
+ 1.to create simple and easy communication between an organized group of people and a supplier.
+2. lower the prices of food cost
+Plan our plan is to create a website that will communicate between the group of people and the supplier, removing the supermarkets from the equation, which will lead to lower the price for the customer.‏
```

Comment:
Isn't the first word "Plan" a section title like "Goals" and "Idea". ?

The description of the plan is not correct (you just repeat the goals). The plan should be about HOW you intend to achieve the goals